doc/simple.md

```diff
@@ -831,3 +832,14 @@ gnuplot.grid(true)
 ```
 ![](image/power.png)
 
+<a name="nn.MM"/>
+## MM ##
+
+`module` = `nn.MM(transA, transB)`
+
+Performs multiplications on one or more pairs of matrices.
+If `transA` is set, the first matrix is transposed before multiplication.
+If `transB` is set, the second matrix is transposed before multiplication.
+By default, the matrices do not get transposed.
+
+The module also accepts 3D inputs which are interpreted as batches of matrices.
```

Comment:
Can you mention what axis order is expected :  A : b x m x n, B: b x n x m. It makes it easier to understand. Maybe add a small example.